packages/react-ui/src/widgets/TableWidgetUI/TableWidgetUI.js

```diff
@@ -57,10 +58,21 @@ function TableWidgetUI({
   height,
   dense,
   isLoading,
-  lastPageTooltip
+  lastPageTooltip,
+  locale
 }) {
   const paginationRef = useRef(null);
 
+  const intl = useImperativeIntl(locale);
+
+  const defaultLabelDisplayedRows = ({ from, to, count }) => {
+    return `${from}–${to} ${intl.formatMessage({ id: 'c4r.widgets.table.of' })} ${
```

Comment:
please create _one_ localized string which is `{from}-{to} of {count} more than {to}` and pass values using `values` mechanism of formatjs/react-intl (used in other parts of PR)

Reply:
Done